integration-templates/zoho-crm/zoho-crm-contacts.ts

```diff
@@ -0,0 +1,134 @@
+import type { ZohoCRMContact, NangoSync } from './models';
+
+export default async function fetchData(nango: NangoSync) {
+    try {
+        const responses = await getAllContacts(nango, '/crm/v2/Contacts');
```

Comment:
I think we need to `batchSave` page per page, to avoid a high memory footprint of the script. Ideally using the `nango.paginate` notation.